std_msgs/Header Header
BlockPixelLoc[] pixel_locs
